### DISCUSSION TITLE
another use for A3 and A4

### DISCUSSION THREAD

**Discussion by sommo** [micbay/RaceTimerAndController#18] [General]
Hi @micbay,
for the one likes me that have only 2 lane, could be possible to use A3 and A4 pin for a different function? like to control a led or other hw?
for example:
A3 and A4 as output for a led, A3 for line1 and A4 for line2, that will flashes on passing. or controlling a (strip) led (relay) that will turn on when the lane is active.

**Comment by micbay** [comment]
I think the best approach is to clarify desired features that people would like added, and if those features require new pins, then those pins can be defined in the `localSettings.h` file. As such, those that only use two lanes could assign pins A3 and A4 to be used for those features, and others could use an Arduino with more pins, and still keep 4 lanes.
For simple LED type features, it's probably best to keep expanding the MAX7219 chain. The same MAX7219 chip used in the DIY start tree, can be wired to drive another 48 LEDs in addition to the start tree LEDs, without needing any new pins.
